.github/workflows/test.yaml

```diff
@@ -138,6 +158,7 @@ jobs:
       - name: Run integration
         # Force one single concurrent test
         run: tox -e integration
+        continue-on-error: true  # so that other juju versions are tested
 
   integration-quarantine:
```

Comment:
I guess we also need `continue-on-error=false` at this level, assuming that having it at this level is correct for `integration`.

Reply:
The commented on region of code that github chooses made this unclear -- it's referring to the `integration-quaraintine` scope.

Reply:
To further attempt to be clear, we have:
`integration.continue-on-error = false`
`integration[Run integration].continue-on-error = true`

`integration-quarantine[Run integration].continue-on-error = true`

And assuming that the setup for `integration` is correct, I think we should also have:
`integration-quarantine.continue-on-error = false`

Reply:
done, thanks